wrapper/src/main/java/software/amazon/jdbc/hostlistprovider/monitoring/MonitoringRdsHostListProvider.java

```diff
@@ -85,17 +85,8 @@ public MonitoringRdsHostListProvider(
         CLUSTER_TOPOLOGY_HIGH_REFRESH_RATE_MS.getLong(this.properties));
   }
 
-  /**
-   * Clear topology cache for the current cluster.
-   */
-  public void clear() {
-    topologyCache.remove(this.clusterId);
-  }
-
   public static void clearCache() {
-    topologyCache.clear();
```

Comment:
intellij was giving me a warning that this implementation is the same as the superclass clearAll implementation